packages/tinacms/src/plugins/fields/BlocksFieldPlugin.tsx

```diff
@@ -99,7 +105,15 @@ const Blocks = function({ tinaForm, form, field, input }: BlockFieldProps) {
   return (
     <>
       <GroupListHeader>
-        <GroupLabel>{field.label || field.name}</GroupLabel>
+        <GroupListMeta>
+          <GroupLabel>{field.label || field.name}</GroupLabel>
+          {field.description && (
+            <FieldDescription>{field.description}</FieldDescription>
+          )}
+        </GroupListMeta>
+        {field.description && (
+          <FieldDescription>{field.description}</FieldDescription>
+        )}
```

Comment:
I'm not really sure what this would look like with the description being rendered twice.

Could you post a screenshot?

Reply:
Actually, since `GroupList` is different I'm guessing that second one shouldn't be there?

Reply:
Fixed.